bower.json

```diff
@@ -5,7 +5,7 @@
         "Zinkerz"
     ],
     "dependencies": {
-        "infra-web-app": "https://github.com/Knowledge-OTP/infra-web-app.git#~v0.0.51"
```

Comment:
Hi,
You already accept infra-web-app version for sat. see closed PR:
https://github.com/Knowledge-OTP/infra-web-app/pull/233